apm-application-toolkit/apm-toolkit-kafka/pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <artifactId>apm-application-toolkit</artifactId>
         <groupId>org.apache.skywalking</groupId>
-        <version>8.13.0-SNAPSHOT</version>
```

Comment:
Why do you change all versions to a release status?

Reply:
I am sorry,  I create branch from tag v8.13.0

Reply:
That is wrong. We only accept PR from a dev version. You need to update from the main branch.

Reply:
ok